[2024-09-18T14:49:15.140+0000] {processor.py:186} INFO - Started process (PID=93976) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:49:15.140+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T14:49:15.140+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:49:15.140+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:49:15.152+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:49:15.152+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T14:49:15.152+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:49:15.152+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T14:49:15.152+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:49:15.179+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-09-18T14:50:35.005+0000] {processor.py:186} INFO - Started process (PID=94214) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:50:35.006+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T14:50:35.006+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:50:35.006+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:50:35.016+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:50:35.016+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T14:50:35.016+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:50:35.016+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T14:50:35.016+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:50:35.040+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-09-18T14:52:14.257+0000] {processor.py:186} INFO - Started process (PID=94445) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:52:14.257+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T14:52:14.258+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:52:14.258+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:52:14.268+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:52:14.268+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T14:52:14.268+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:52:14.268+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T14:52:14.268+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:52:14.291+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-09-18T14:53:53.830+0000] {processor.py:186} INFO - Started process (PID=94614) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:53:53.831+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T14:53:53.832+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:53:53.832+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:53:53.842+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:53:53.841+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T14:53:53.842+0000] {logging_mixin.py:190} INFO - [2024-09-18T14:53:53.842+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T14:53:53.842+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T14:53:53.864+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2024-09-18T15:21:35.316+0000] {processor.py:186} INFO - Started process (PID=97179) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:21:35.317+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:21:35.317+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:21:35.317+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:21:35.327+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:21:35.326+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:21:35.327+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:21:35.327+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:21:35.327+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:21:35.349+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2024-09-18T15:23:13.750+0000] {processor.py:186} INFO - Started process (PID=97354) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:23:13.751+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:23:13.751+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:23:13.751+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:23:13.761+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:23:13.761+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:23:13.761+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:23:13.761+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:23:13.761+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:23:13.783+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2024-09-18T15:27:18.657+0000] {processor.py:186} INFO - Started process (PID=97811) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:27:18.658+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:27:18.658+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:27:18.658+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:27:18.668+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:27:18.668+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:27:18.668+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:27:18.668+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:27:18.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:27:18.692+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-09-18T15:28:38.504+0000] {processor.py:186} INFO - Started process (PID=98010) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:28:38.505+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:28:38.506+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:28:38.506+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:28:38.516+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:28:38.516+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:28:38.516+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:28:38.516+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:28:38.516+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:28:38.540+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-09-18T15:30:17.506+0000] {processor.py:186} INFO - Started process (PID=98287) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:30:17.507+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:30:17.508+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:30:17.508+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:30:17.518+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:30:17.518+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:30:17.518+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:30:17.518+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:30:17.518+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:30:17.541+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-09-18T15:31:53.000+0000] {processor.py:186} INFO - Started process (PID=98570) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:31:53.000+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:31:53.001+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:31:53.001+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:31:53.011+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:31:53.011+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:31:53.011+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:31:53.011+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:31:53.011+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:31:53.034+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-09-18T15:34:55.984+0000] {processor.py:186} INFO - Started process (PID=99011) to work on /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:34:55.985+0000] {processor.py:914} INFO - Processing file /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-18T15:34:55.985+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:34:55.985+0000] {dagbag.py:587} INFO - Filling up the DagBag from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:34:55.995+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:34:55.995+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-18T15:34:55.995+0000] {logging_mixin.py:190} INFO - [2024-09-18T15:34:55.995+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-18T15:34:55.995+0000] {processor.py:927} WARNING - No viable dags retrieved from /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-18T15:34:56.018+0000] {processor.py:208} INFO - Processing /Users/omer/Desktop/MLOPS/7day/.env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
